libs/core/src/components/pds-avatar/pds-avatar.tsx

```diff
@@ -72,8 +72,8 @@ export class PdsAvatar {
 
   private renderAvatar = () => {
     const style = {
-      height: this.avatarSize(),
-      width: this.avatarSize(),
+      height: this.badge ? `${parseInt(this.avatarSize()) + 10}px` : this.avatarSize(),
+      width: this.badge ? `${parseInt(this.avatarSize()) + 10}px` : this.avatarSize()
```

Comment:
I am not positive we need the additional 10px?

Reply:
we don't need this. So a good catch. This was one of the attempts to solve it differently.